### DISCUSSION TITLE
A Mac M1 2020 user cannot use Freetube?

### DISCUSSION THREAD

**Discussion by Aslinjean** [FreeTubeApp/FreeTube#5565] [General]
Hello team
Every time I install FreeTube on my 2020 M1 Mac, Apple tells me it's dangerous for my device and automatically refuses the application. So, can someone tell me what I need to do to successfully download FreeTube on my device? Please!

**Comment by avlknh** [comment]
open it via right click the freetube app on your finder

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
fixed in https://github.com/FreeTubeApp/FreeTube/discussions/5516#discussioncomment-10322513
